RFC/src/RFC-0120_Consensus.md

```diff
@@ -248,27 +290,75 @@ The [PoW] MUST contain the following:
 * pow_algo as an enum (0 for monero, 1 for blake).
 * pow_data as array of unsigned 8-bit integers (bytes) in little-endian format.
 
+#### Difficulty Calculation
+[target difficulty]: #target-difficulty "Target Difficulty"
+
+The target difficulty represents how difficult it is to mine a given block. This difficulty is not fixed and needs to
+constantly adjust to changing network hash rates. 
+
+The difficulty adjustment MUST be calculated using a Linear-Weighted moving average (LWMA) algorithm
+
+$$
+\begin{align}
+\\ N \text{ target difficulty window}
```

Comment:
we should add the actual numbers here, doubt if they are going to change between now and main-net

Do you think we should mention the "fixing" of negative timestamps?

Reply:
For completeness, yeah I think we should. Agreed, these are not likely to change. I'll update with actual numbers.

Reply:
Done